src/plugins/inputs/ignore.js

```diff
@@ -110,35 +111,41 @@ exports.input = function (network, chan, cmd, args) {
 					})
 				);
 			} else {
-				const chanName = "Ignored users";
-				const ignored = network.ignoreList.map((data) => ({
-					hostmask: `${data.nick}!${data.ident}@${data.hostname}`,
-					when: data.when,
-				}));
-				let newChan = network.getChannel(chanName);
-
-				if (typeof newChan === "undefined") {
-					newChan = client.createChannel({
-						type: Chan.Type.SPECIAL,
-						special: Chan.SpecialType.IGNORELIST,
-						name: chanName,
-						data: ignored,
-					});
-					client.emit("join", {
-						network: network.uuid,
-						chan: newChan.getFilteredClone(true),
-						index: network.addChannel(newChan),
-					});
-				} else {
-					newChan.data = ignored;
-
-					client.emit("msg:special", {
-						chan: newChan.id,
-						data: ignored,
-					});
-				}
+				create_new_ignored_window = true;
 			}
 
 			break;
 	}
+
+	const chanName = "Ignored users";
+	const channel = network.getChannel(chanName);
+
+	if (typeof channel === "undefined" && !create_new_ignored_window) {
```

Comment:
The only point of this flag is to get rid of ignore + unignore again, you wouldn't even have to have this flag if you scope it properly.